src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -599,14 +528,13 @@ void EnGo_DrawDust(EnGoEffect effect[ENGO_EFFECT_COUNT], PlayState* play2) {
  * @param effect The EnGoEffect table
  * @param pos Position around which the effects appear
  */
-void EnGo_InitSnow(EnGoEffect effect[ENGO_EFFECT_COUNT], Vec3f pos) {
+void EnGo_InitSnow(EnGoEffect effect[ENGO_SNOW_EFFECT_COUNT], Vec3f pos) {
     static u8 effectIndexToSnowEffectTable[] = {
```

Comment:
Should this (and others) specify array size too?

Reply:
This table makes sense, but I'm not seeing any other tables at a quick glance

Reply:
@EllipticEllipsis bump to make sure you saw this.